File: SOKOBAN/src/code/objet.java
package code;

public class objet {
	
    int m_x, m_y;
    
    public objet(int x, int y) {
    	m_x = x;
    	m_y = y;
    }
    public int getX(){
        return m_x;
    }

    public int getY(){
        return m_y;
    }

    public void setX(int newX){
        m_x = newX;
    }

    public void setY(int newY){
        m_y = newY;
    }
}

File: MonSokoban/objet.cpp
#include "objet.h"

Objet::Objet(int x, int y){
    m_x = x;
    m_y = y;
}

int Objet::getX(){
    return m_x;
}

int Objet::getY(){
    return m_y;
}

void Objet::setX(int newX){
    m_x = newX;
}

void Objet::setY(int newY){
    m_y = newY;
}
